docs/DeveloperGuide.md

```diff
@@ -808,7 +837,7 @@ testers are expected to do more *exploratory* testing.
 
    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
 
-   2. Test case: `delete alexy@u.nus.edu`<br>
+   2. Test case: `delete alexyeoh@u.nus.edu`<br>
       Expected: Contact with above email deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
```

Comment:
Can change the parts that mention contact to student pls

Reply:
Okay